halo2-repl/app/worker/halo2repl.ts

```diff
@@ -4,6 +4,11 @@ import { Halo2CircuitRunner, captureConsoleOutput } from '@axiom-crypto/halo2-li
 import { Halo2Lib } from '@axiom-crypto/halo2-lib-js';
 import { Halo2LibWasm, getHalo2LibWasm, CircuitScaffold, getKzgParams, DEFAULT_CIRCUIT_CONFIG } from '@axiom-crypto/halo2-wasm/web'
 
+const DEFAULT_HALO2REPL_CONFIG = {
+    ... DEFAULT_CIRCUIT_CONFIG,
+    k: 20,
```

Comment:
I think `k = 20` is overkill and too large for most use cases. You'd only use this is you needed ecc. Also we should just increase the default columns rather than `k`.

Reply:
Likely keeping `k = 14` (but perhaps with more columns) should be enough for default things.